### PR TITLE
Simplify the submission form by removing ad and event options for new…

### PR DESCRIPTION
**Pull request by ericras**
… items. Retain backwards compatibility with existing items

### CODE REVIEW

**Comment by ericras** [comment]
I think this all is basically ready to look at: http://ucommrasmussen.unl.edu/workspace/UNL_ENews/www/

**Comment by jsturek** [comment]
- [ ] On `Submit an Item` page need to add margin-bottom to submit button so displays better in mobile.

File: www/css/manage.css [hunk 1]
```diff
@@ -65,10 +33,10 @@ form fieldset.storyFieldsetAction select {
 }
 .storylisting.functionTable tbody tr.event td:first-of-type {
     background:url('images/typeIndicatorEvent.png') no-repeat 0 0;
-}
+} /* No longer show News banner since every item is now news
```
Comment (jsturek) [inline www/css/manage.css]: Should we just remove instead of commenting out

**Comment by jsturek** [comment]
Looks good other the above suggestions.  You can merge when and if addressed.

File: www/css/manage.css (hunk 1 above)
Comment (ericras) [reply www/css/manage.css]: I thought it might be helpful if someone goes looking for why there are only two types of banners and not a third

**Pull request merged**